chat.go

```diff
@@ -2,8 +2,81 @@ package statusproto
 
 import "crypto/ecdsa"
 
-type Chat interface {
-	ID() string
-	PublicName() string
-	PublicKey() *ecdsa.PublicKey
+type ChatPagination struct {
+	From uint
+	To   uint
+}
+
+type ChatType int
+
+const (
+	ChatTypeOneToOne = iota
```

Comment:
```suggestion
	ChatTypeOneToOne = iota + 1
```

to avoid confusion of `ChatType` zero-value and `ChatTypeOneToOne`.

Unless status-react requires it to be `0`.